contrib/rofi/template.py

```diff
@@ -0,0 +1,45 @@
+pkgname = "rofi"
+pkgver = "1.7.5"
+_release = 2
+pkgrel = 0
```

Comment:
hiding the release out of pkgver is wrong, because there are actually multiple of those. e.g. there was 1.7.5+wayland1 to 1.7.5+wayland2. you can't paper over that with "1.7.5 and then increment the pkgrel".

i would probably pick `1.7.5_p2`

Reply:
ufff okay